src/connection_string.ts

```diff
@@ -1224,8 +1224,11 @@ export const OPTIONS = {
       'useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version'
   } as OptionDescriptor,
   // MongoLogger
-  // TODO(NODE-4849): Tighten the type of mongodbLogPath
-  mongodbLogPath: { type: 'any' }
```

Comment:
we still need to be able to specify mongodbLogPath in the connection string, so I don't think we can remove it from here;

we also need to make sure to add tests for specifying this option both in the connection string and as a constructor option

Reply:
I'll fix this in my next commit, and unnest the option